### PR TITLE
Dịch "dropout_vn" - Phần 1

### LINKED ISSUE #943: Dịch "dropout_vn" - Phần 1
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/dropout_vn.md#L1-L70

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by lkhphuc**
Nhận xong quên mất tiêu.
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #943 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 1]
```diff
@@ -46,15 +53,20 @@ They can learn interactions among groups of features.
 For example, they might infer that “Nigeria” and “Western Union” appearing together in an email indicates spam but that separately they do not.
 -->
 
-*dịch đoạn phía trên*
+Mạng nơ-ron sâu lại nằm ở thái cực trái ngược trên phổ độ chệch - phương sai.
+Không giống mô hình tuyến tính, các mạng nơ-ron không bị giới hạn chỉ được thấy từng đặc trưng riêng biệt.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Không giống mô hình tuyến tính, các mạng nơ-ron không bị giới hạn ở việc chỉ được xét từng đặc trưng riêng biệt.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 2]
```diff
@@ -46,15 +53,20 @@ They can learn interactions among groups of features.
...
+Chẳng hạn, chúng có thể suy ra được rằng nếu từ "Nigeria" và "Western Union" xuất hiện cùng nhau trong một email thì đó là thư rác, nhưng nếu hai từ đó xuất hiện riêng biệt thì không phải.
 
 <!--
 Even when we have far more examples than features, deep neural networks are capable of overfitting.
 In 2017, a group of researchers demonstrated the extreme flexibility of neural networks by training deep nets on randomly-labeled images.
 Despite the absence of any true pattern linking the inputs to the outputs, they found that the neural network optimized by SGD, could label every image in the training set perfectly.
 -->
 
-*dịch đoạn phía trên*
+Ngay cả khi số mẫu ta có hơn rất nhiều so với số đặc trưng, mạng nơ-ron sâu vẫn có thể quá khớp.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Ngay cả khi số mẫu nhiều hơn hẳn so với số đặc trưng, mạng nơ-ron sâu vẫn có thể quá khớp.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 3]
```diff
@@ -46,15 +53,20 @@ They can learn interactions among groups of features.
...
+Năm 2017, một nhóm nhà nghiên cứu đã minh chứng khả năng linh hoạt tột cùng của mạng nơ-ron bằng cách huấn luyện một mạng sâu trên tập ảnh được gán nhãn ngẫu nhiên.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Năm 2017, một nhóm các nhà nghiên cứu đã minh chứng khả năng linh hoạt tột cùng của mạng nơ-ron bằng cách huấn luyện mạng nơ-ron sâu trên tập ảnh được gán nhãn ngẫu nhiên.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 4]
```diff
@@ -46,15 +53,20 @@ They can learn interactions among groups of features.
...
+Mặc cho không hề có bất cứ một khuôn mẫu nào liên kết đầu vào và đầu ra, họ phát hiện rằng mạng nơ-ron được tối ưu bằng SGD vẫn có thể khớp tất cả nhãn trên tập huấn luyện một cách hoàn hảo.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Dù không hề có bất cứ một khuôn mẫu nào liên kết đầu vào và đầu ra, họ phát hiện rằng mạng nơ-ron được tối ưu bằng SGD vẫn có thể khớp tất cả nhãn trên tập huấn luyện một cách hoàn hảo.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 5]
```diff
@@ -65,7 +77,12 @@ The mathemtatical foundations for the puzzling generalization properties of deep
 For now, we turn to the more terrestrial investigation of practical tools that tend (empirically) to improve the generalization of deep nets.
 -->
 
-*dịch đoạn phía trên*
+Thử nghĩ xem điều này hàm chứa ý nghĩa gì.
+Nếu nhãn được gán ngẫu nhiên từ một phân phối đều với 10 lớp, sẽ không có bộ phân loại nào có thể có độ chính xác cao hơn 10% trên tập dữ liệu kiểm tra.
+Khoảng cách khái quát là tận 90%.
+Nếu mô hình của chúng ta có đủ năng lực để quá khớp tới như vậy, phải như thế nào chúng ta mới có thể trông đợi rằng mô hình sẽ không quá khớp?
+Nền tảng toán học đằng sau tính chất khái quát hoá hóc búa của các mạng sâu này vẫn còn là một câu hỏi mở, và chúng tôi khuyến khích bạn đọc có khuyên hướng lý thuyết hãy đào sâu hơn vào chủ đề này.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]:
```suggestion
Nền tảng toán học đằng sau tính chất khái quát hoá hóc búa của mạng nơ-ron sâu vẫn còn là một câu hỏi mở, và chúng tôi khuyến khích bạn đọc có khuynh hướng lý thuyết đào sâu hơn vào chủ đề này.
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 6]
```diff
@@ -65,7 +77,12 @@ The mathemtatical foundations for the puzzling generalization properties of deep
...
+Còn bây giờ ở trên bề mặt, chúng ta chuyển sang tìm hiểu về các công cụ để cải thiện khả năng khái quát của các mạng sâu theo thực nghiệm.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/dropout_vn.md]: ông tác giả thích dùng từ hoa mỹ vãi :))
```suggestion
Còn bây giờ hãy quay về bề mặt của vấn đề và chuyển sang tìm hiểu các công cụ để cải thiện khả năng khái quát của các mạng nơ-ron sâu dựa trên thực nghiệm. 
```

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 7]
```diff
@@ -30,14 +32,19 @@ Naively applied, linear models do not take into account interactions among featu
 For every feature, a linear model must assign either a positive or a negative weight, ignoring context.
 -->
 
-*dịch đoạn phía trên*
+Có nhiều đặc trưng hơn số mẫu, các mô hình tuyến tính có xu hướng sẽ quá khớp.
+Tuy nhiên nếu có nhiều mẫu hơn số đặc trưng, nhìn chung ta có thể tin cậy mô hình tuyến tính sẽ không quá khớp.
```
[the remark this answers is not in the document]
Comment (lkhphuc) [reply chapter_multilayer-perceptrons/dropout_vn.md]: `tin rằng` không sát nghĩa `count on` mà giống `believe`.

File: chapter_multilayer-perceptrons/dropout_vn.md [hunk 8]
```diff
@@ -30,14 +32,19 @@ Naively applied, linear models do not take into account interactions among featu
...
+Thật không may, mô hình tuyến tính dựa trên tính ổn định này để khái quát hoá lại kèm theo một cái giá phải trả là:
```
[the remark this answers is not in the document]
Comment (lkhphuc) [reply chapter_multilayer-perceptrons/dropout_vn.md]: Câu gốc mình hiểu là "mô hình tuyến tính khái quát được là dựa trên tính ổn định của nó", nên nếu sửa như vậy thì không đúng.

[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: `reliable` cũng có nghĩa là `đáng tin cậy` mà, nên mình hiểu là tác giả đang kết nối với câu trước

[the remark this answers is not in the document]
Comment (lkhphuc) [reply chapter_multilayer-perceptrons/dropout_vn.md]: Hmm, chắc mình hiểu khác nhau. Nếu dịch như @minhduc0711 thì câu gốc mình thấy phải là:
`...the reliability **that** linear models generalize ...`

[the remark this answers is not in the document]
Comment (minhduc0711) [reply chapter_multilayer-perceptrons/dropout_vn.md]: Mình lại không tin tưởng vào phrasing của tác giả lắm :smile: 
Thôi tạm thời cứ để nguyên ban đầu đi vầy
